src/main/scala/org/ergoplatform/nodeView/ErgoReadersHolder.scala

```diff
@@ -32,7 +32,10 @@ class ErgoReadersHolder(viewHolderRef: ActorRef) extends Actor with ScorexLoggin
       mempoolReaderOpt = Some(reader)
 
     case GetReaders =>
-      sender ! Readers(historyReaderOpt, stateReaderOpt, mempoolReaderOpt)
+      (historyReaderOpt, stateReaderOpt, mempoolReaderOpt) match {
+        case (Some(h), Some(s), Some(m)) => sender ! Readers(h, s, m)
+        case _ =>
```

Comment:
if we use `?` pattern here to get the readers, then the caller will wait the whole timeout, right?

Reply:
Maybe we need a comment here that we never have Nones here after successful node start